invokeai/backend/util/hotfixes.py

```diff
@@ -745,7 +747,9 @@ def forward(
             scales = torch.logspace(-1, 0, len(down_block_res_samples) + 1, device=sample.device)  # 0.1 to 1.0
 
             scales = scales * conditioning_scale
-            down_block_res_samples = [sample * scale for sample, scale in zip(down_block_res_samples, scales)]
+            down_block_res_samples = [
+                sample * scale for sample, scale in zip(down_block_res_samples, scales, strict=True)
+            ]
```

Comment:
When `zip` has receives collections of different lengths, it silently truncates the collections to the length of the shortest collection.

This is likely not intended, so `ruff` wants us to always specify the `strict` arg. 

If `strict=True`, and the collections are of different length, an error is raised. If `strict=False`, you get the silent truncation behaviour.